tests/cases/standalone/common/alter/change_col_inverted_index.result

```diff
@@ -0,0 +1,183 @@
+| test_pk | 1          | PRIMARY, INVERTED INDEX | 3            | bar         | A         |             |          |        | YES  | greptime-primary-key-v1, greptime-inverted-index-v1 |         |               | YES     |            |
+| test_pk | 1          | PRIMARY, INVERTED INDEX | 2            | foo         | A         |             |          |        | YES  | greptime-primary-key-v1, greptime-inverted-index-v1 |         |               | YES     |            |
+| test_pk | 1          | TIME INDEX              | 1            | ts          | A         |             |          |        | NO   |                                                     |         |               | YES     |            |
++---------+------------+-------------------------+--------------+-------------+-----------+-------------+----------+--------+------+-----------------------------------------------------+---------+---------------+---------+------------+
+
+ALTER TABLE test_pk MODIFY COLUMN foo UNSET INVERTED INDEX;
+
+Affected Rows: 0
+
+SHOW INDEX FROM test_pk;
+
++---------+------------+-------------------------+--------------+-------------+-----------+-------------+----------+--------+------+-----------------------------------------------------+---------+---------------+---------+------------+
+| Table   | Non_unique | Key_name                | Seq_in_index | Column_name | Collation | Cardinality | Sub_part | Packed | Null | Index_type                                          | Comment | Index_comment | Visible | Expression |
++---------+------------+-------------------------+--------------+-------------+-----------+-------------+----------+--------+------+-----------------------------------------------------+---------+---------------+---------+------------+
+| test_pk | 1          | PRIMARY, INVERTED INDEX | 3            | bar         | A         |             |          |        | YES  | greptime-primary-key-v1, greptime-inverted-index-v1 |         |               | YES     |            |
+| test_pk | 1          | PRIMARY, INVERTED INDEX | 2            | foo         | A         |             |          |        | YES  | greptime-primary-key-v1, greptime-inverted-index-v1 |         |               | YES     |            |
+| test_pk | 1          | TIME INDEX              | 1            | ts          | A         |             |          |        | NO   |                                                     |         |               | YES     |            |
++---------+------------+-------------------------+--------------+-------------+-----------+-------------+----------+--------+------+-----------------------------------------------------+---------+---------------+---------+------------+
+
+ALTER TABLE test_pk MODIFY COLUMN bar UNSET INVERTED INDEX;
+
+Affected Rows: 0
+
+SHOW INDEX FROM test_pk;
```

Comment:
not sure if this is the result we are expecting, nothing changes when we set/ unset inverted index on primary keys that comes with inverted index.

Reply:
The table has two primary keys with no explicit inverted index, so `is_inverted_indexed` and `has_inverted_index_key` will return false. So that `SHOW INDEX` finds no inverted index in the table and add primary keys into the inverted index sets, see:
https://github.com/GreptimeTeam/greptimedb/blob/812a775b3d02da1430364dfa7ef14d31bfab800a/src/catalog/src/system_schema/information_schema/key_column_usage.rs#L231-L234

Seems we need to set `inverted_indexed=true` for column `bar` when we unset inverted index for column `foo` to get the expected result. When it comes to unset inverted index in all columns, seems it's impossible? cc @zhongzc

Reply:
> Seems we need to set inverted_indexed=true for column bar when we unset inverted index for column foo to get the expected result. 
thanks for pointing out the logic, i had a follow up when we remove foo in the next iteration there will be no inverted index on the table - do we expecting primary key added as inverted index again?

Reply:
In this case, we expect all primary key shows no inverted index in the query, ref https://github.com/GreptimeTeam/greptimedb/pull/5131#discussion_r1906577069:
```
CREATE TABLE test_pk (..., foo STRING, bar INT, PRIMARY KEY (foo, bar));
SHOW INDEX; // inverted indexes: [foo, bar]

UNSET foo;
SHOW INDEX; // inverted indexes: [bar]

UNSET bar;
SHOW INDEX; // inverted indexes: []

SET foo;
SHOW INDEX; // inverted indexes: [foo]
```

There're two cases:
1. All primary key are marked non-inverted index but we implicitly build inverted index for primary keys. We should show them as inverted index columns.
2. All primary key are marked non-inverted index, and it's explicitly unset by users. We don't need to show them as inverted index columns.

Reply:
thanks i missed the expected result from previous comment. I added logic to make the results converge but the logic is not straight forward :(